pkg/util/addrutil/addrutil.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package addrutil
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+)
+
+// IoAddrToEvmAddr converts IoTeX address into evm address
+func IoAddrToEvmAddr(ioAddr string) (common.Address, error) {
+	address, err := address.FromString(ioAddr)
+	if err != nil {
+		return common.Address{}, output.NewError(output.ConvertError, "", err)
```

Comment:
This pkg should not depend on the package from ioctl

Reply:
+1

Reply:
fix this issue before merging

Reply:
ok